api/v1alpha1/virtualmachine_conversion_test.go

```diff
@@ -704,6 +713,313 @@ func TestVirtualMachineConversion(t *testing.T) {
 			g.Expect(c.Reason).To(Equal("Reason"))
 			g.Expect(c.Message).To(Equal("Message"))
 		})
+
+		t.Run("Converts PrereqReady Condition", func(t *testing.T) {
+			t.Run("PrereqReady is True with no prior v1a2 conditions", func(t *testing.T) {
+				g := NewWithT(t)
+
+				now := metav1.Now()
+				spoke := v1alpha1.VirtualMachine{
+					Spec: v1alpha1.VirtualMachineSpec{
+						VmMetadata: &v1alpha1.VirtualMachineMetadata{
```

Comment:
Can we also have a test w/o the `VmMetadata` and `ReasourcePolicyName` set? I'd like to have something to ensure that we don't NPE.